hasura/migrations/default/1668026379094_alter_table_public_epoches_add_column_description/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."epoches" add column "description" varchar
```

Comment:
why not use a `text` type ?

Reply:
@zashton  I set the maximum to 100 characters so far in the form. should I change it to text?

Reply:
@sshmm there is no semantic difference under the hood between `varchar` and `text`. This is just for consistency.